core/src/test/java/io/grpc/internal/SpiffeUtilTest.java

```diff
@@ -159,7 +170,8 @@ public void spiffeTrustDomainFormatTest() {
           SpiffeUtil.parse(longTrustDomain.toString()));
       assertEquals("Trust Domain maximum length is 255 characters", iae.getMessage());
 
-      StringBuilder longSpiffe = new StringBuilder(String.format("spiffe://mydomain%scom/", "%21"));
+      @SuppressWarnings("OrphanedFormatString")
+      StringBuilder longSpiffe = new StringBuilder("spiffe://mydomain%21com/");
```

Comment:
@ejona86 
The situation with that string is more interesting then we thought - actually they don't show warning for `https`. Filed a bug - https://github.com/google/error-prone/issues/4609

Reply:
Okay. Yeah, let's just suppress it for now. It produces harder-to-read code if trying to work around it in other ways.